operatorcourier/manifest_parser.py

```diff
@@ -0,0 +1,82 @@
+import os
+import logging
+from operatorcourier.errors import OpCourierBadBundle
+from operatorcourier import identify
```

Comment:
in `flatten.py` you refactored this import:

```diff
-from operatorcourier import identify
+from operatorcourier.identify import get_operator_artifact_type
```

and made review harder by this non-relevant change. And suddenly here you use inconsistently the same import you refactored in previous files. So when you refactor keep it consistent.

Reply:
@MartinBasti Changed `from operatorcourier.identify import get_operator_artifact_type` back to `from operatorcourier import identify`.